### DISCUSSION TITLE
Remove outro in trailer

### DISCUSSION THREAD

**Discussion by Hello-World-Traveler** [nandyalu/trailarr#75] [General]
@nandyalu It's nice to see this project taking shape.

How does trailarr deal with outros after the trailer has finished? Some videos have that "watch more" area before the end.

**Comment by nandyalu** [comment]
I did notice that most videos will have an outro, and I thought it would be better to remove them.

SponsorBlocks was the initial solution I found, but most trailers don't have any SponsorBlocks info, so that didn't work. 

I am working on a solution that uses ffmpeg to look for empty audio at the end of the video and remove that. It's still in the works. If you're interested, get on over to Discord, and we can discuss more.

**Reply by Hello-World-Traveler** [reply]
The last file I downloaded included music at the end, so relying on empty audio might not always be effective. 
I’ll look into getting Discord.
